hl7v2_to_fhir/.devcontainer.json

```diff
@@ -0,0 +1,4 @@
+{
+    "image": "ballerina/ballerina-devcontainer:2201.4.2",
```

Comment:
I think Choreo is supporting version 5, so we can update the version here?

Reply:
this is auto-gen. Let me remove this file.